File: Week 6/sentimental-mario-less/mario.py
# TODO
import cs50

# prompt user for int 1-8
while True:
    Height = cs50.get_int("Enter Height here: ")
    if Height >= 1 and Height <= 8:
        break

# row height outer
for row in range(Height):
    for space in range(Height):
        if (row + space >= Height - 1):
            print("#", end="")
        else:
            print(" ", end="")
    print()

File: Week 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, row, column, space;
    // int prompt between 1 - 8
    do
    {
        height = get_int("Enter Height here: ");
    }
    while (height < 1 || height > 8);

    // row height outer loop
    for (row = 0; row < height; row++)
    {
        //spaces loop
        for (space = 0; space < height - row - 1; space++)
        {
            printf(" ");
        }
        //column width inner loop
        for (column = 0; column <= row; column++)
        {
            printf("#");
        }
        printf("\n");
    }
}